cmd/list.go

```diff
@@ -283,6 +327,15 @@ func (cooked cookedListCmdArgs) HandleListContainerCommand() (err error) {
 	}
 
 	if cooked.RunningTally {
+		if containsProperty(cooked.properties, versionId) {
+			// get file count by getting length of objectVer
+			fileCount = int64(len(objectVer))
+
+			// get size count by incrementing through objectVer and adding all values of versionIdObject.fileSize
+			for _, val := range objectVer {
```

Comment:
this could result in possibly looping over ever blob 2 times. Can we make the code above smarter and update fileCount + sizeCount dynamically instead of looping twice